crates/wast/src/ast/expr.rs

```diff
@@ -1144,6 +1144,9 @@ instructions! {
         F32x4RelaxedMax : [0xfd, 0xe2]: "f32x4.relaxed_max",
         F64x2RelaxedMin : [0xfd, 0xd4]: "f64x2.relaxed_min",
         F64x2RelaxedMax : [0xfd, 0xee]: "f64x2.relaxed_max",
+
+        CanonLift(CanonLift<'a>) : [/*TODO: encoding*/]: "canon.lift",
+        CanonLower(CanonLower<'a>) : [/*TODO: encoding*/]: "canon.lower",
```

Comment:
Given how separate these are from all the other instructions I think it'd be better to not put this in the instruction enum and have a dedicated enum for component-defined functions (e.g. `ComponentFunc`)